lte/gateway/python/scripts/dhcp_helper_cli.py

```diff
@@ -0,0 +1,352 @@
+#!/usr/bin/env python3
+
+"""
+Copyright (C) 2022  The Magma Authors
+
+This program is free software; you can redistribute it and/or
+modify it under the terms of the GNU General Public License
+as published by the Free Software Foundation; either version 2
+of the License, or (at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program; If not, see <http://www.gnu.org/licenses/>.
+"""
+
+import argparse
+import json
+import time
+from datetime import datetime, timedelta
+from enum import IntEnum
+from queue import Empty, Queue
+from typing import Any, Callable, Dict, List, Optional, Tuple, Union
+
+import scapy.packet
+from scapy.all import AsyncSniffer
+from scapy.layers.dhcp import BOOTP, DHCP
+from scapy.layers.inet import IP, UDP
+from scapy.layers.l2 import Dot1Q, Ether
+from scapy.sendrecv import sendp
+
+
+class DHCPState(IntEnum):
+    UNKNOWN = 0
+    DISCOVER = 1
+    OFFER = 2
+    REQUEST = 3
+    DECLINE = 4
+    ACK = 5
+    NAK = 6
+    RELEASE = 7
+    FORCE_RENEW = 8
+
+
+class MacAddress:
+    def __init__(self, mac: str):
+        self.mac_address = mac.lower()
+
+    def __eq__(self, other):
+        return hasattr(other, 'mac_address') and self.mac_address == other.mac_address
```

Comment:
We could just use `return self.mac_address == getattr(other, "mac_address", "some_default")` or something similar.

Reply:
This would be  equivillant but more implicit I guess - I would like to keep it this way.